automated-demo/yaml-resources/kubefed-operator/01-catalog-source-config.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: operators.coreos.com/v1
 kind: CatalogSourceConfig
 metadata:
-  name: installed-federation-changemeNS
+  name: installed-federation-kube-federation-system
```

Comment:
As in the script, I'd keep the `changemeNS` reference and update the files with sed in the script. That way if the ns name changes in the future we don't need to update every reference to the namespace name.